### DISCUSSION TITLE
Can I "joke" participants?

### DISCUSSION THREAD

**Discussion by dasaz** [jspsych/jsPsych#830] [Q&A] [answered]
I have a study where I randomly assign stimuli to condition for every odd subject and then joke every even subject to them, so they see the same stimuli but with reversed assignment to conditions. I do this in Matlab by initializing the random number generator seed to sub_number/2.
Running studies online, I don't seem to have control over subject number, neither I seem to have control over the random number generator seed. Is there another way how can I balance/yoke assignment of stimuli to conditions or is random assignment my only option?

**Comment by jkhartshorne** [comment, marked as answer]
@dasaz that's a really good question. We're discussing it to see if we've got a solution using pure jsPsych. You can definitely do it, but maybe not with jsPsych alone. We're thinking.

**Reply by jkhartshorne** [reply]
So this isn't something you can do purely in browser. You need some kind of [back-end](https://learntocodewith.me/posts/backend-development/). jsPsych is purely front-end. There are a number of back-end systems out there that work with jsPsych (see #815), and we'll be talking about one of them in Week 4. Many of them would probably allow you to use the solution you mention above: use a random seed based on subject number. With PsiTurk, you could definitely just generate all your stimulus lists in advance and it would automatically assign the right number of subjects to each. 

But I would also say that if you are using yoking in order to make sure you have balanced stimuli and conditions, that's mainly an issue with small numbers of subjects and becomes less important if you are testing large numbers.
